### PR TITLE
add linux conntrack metrics

### PR DESCRIPTION
**Pull request by emilgelman**
## Changes

Suggestion to add two network statistic metrics for Linux connection tracking, taken from https://manpages.debian.org/testing/conntrack/conntrack.8.en.html
Implementation PR: open-telemetry/opentelemetry-collector#3370

### CODE REVIEW

**Comment by emilgelman** [comment]
@jmacd reopened this due to EasyCLA issue

**Comment by SergeyKanzhelev** [comment]
@open-telemetry/specs-metrics-approvers any takers on reviewing this?

**Comment by SergeyKanzhelev** [comment]
Hint for reviewers - ignore whitespaces:

image

File: specification/metrics/semantic_conventions/system-metrics.md [hunk 1]
```diff
@@ -110,19 +110,22 @@ perf counter (similar for Writes)
...
+| system.network.connections                          |                                                                               | {connections} | UpDownSumObserver  | Int64      | device        | (identifier)                                                                                   |
+|                                                     |                                                                               |               |                    |            | protocol      | tcp, udp, [etc.](https://en.wikipedia.org/wiki/Transport_layer#Protocols)                      |
+|                                                     |                                                                               |               |                    |            | state         | [e.g. for tcp](https://en.wikipedia.org/wiki/Transmission_Control_Protocol#Protocol_operation) |
+| system.connections.table.usage<sup>[3](#linux)</sup>| Number of currently allocated flow entries                                    | {flow entries} | UpDownSumObserver | Int64      |               |                                                                                                |
```
Comment (jsuereth) [inline specification/metrics/semantic_conventions/system-metrics.md:126]: Two questions pop to mind:

1. Can you link to an article describing what "flow entries" mean, or denoting what / how this is monitoring linux firewalls.
2. Should there be any labels associated with data extracted from conntrack?   Similar to what you see above, e.g. tcp vs udp, ESTABLISHED vs TIME_WAIT, etc.

Also how does this differ from `system.network.connections` besides being a view from the firewall (IIUC)?

Comment (jmacd) [reply specification/metrics/semantic_conventions/system-metrics.md:126]: Good questions. I had recommended the use of ".limit" and ".usage" conventions, but the prefix "system.connections.table" could perhaps be more descriptive. (See https://github.com/open-telemetry/opentelemetry-specification/pull/1761#discussion_r663227318)

**Pull request closed without merging**